inline.h

```diff
@@ -3624,6 +3624,48 @@ Perl_savesharedsvpv(pTHX_ SV *sv)
     return savesharedpvn(pv, len);
 }
 
+#ifndef PERL_GET_CONTEXT_DEFINED
+
+/*
+=for apidoc_section $embedding
+=for apidoc get_context
+
+Implements L<perlapi/C<PERL_GET_CONTEXT>>, which you should use instead.
```

Comment:
> ..\av.c(1208): error C2129: static function 'void *Perl_get_context(void)' declared but not defined

The version from `win32/win32thread.c` should probably be inlined too, possibly in `win32/win32thread.h`